appinventor/components/src/com/google/appinventor/components/runtime/Sprite.java

```diff
@@ -332,8 +366,6 @@ public double Y() {
    *        in front of ones with lower numbers; if values are equal for
    *        sprites, either can go in front of the other
    */
-  @SimpleProperty(
```

Comment:
By removing this `@SimpleProperty` the `set Z` property block disappears in the language.

Reply:
Yikes!

Reply:
@ewpatton I put that back in, but I'm still having a (possibly unrelated) problem with the Z property in the Designer view. No value is being set for Z. I could use another pair of eyes if you wouldn't mind helping me figure out what I did wrong.

Reply:
Actually, hang on a minute. I'm wondering if it has to do with the calls to `reorderComponents()` that I thought were superfluous.

Reply:
No, that didn't make a difference. I would welcome your help when you get a chance.

Reply:
@espertus I believe the issue with `Z` in the designer is due to the introduction of `@DesignerProperty` on line 381, which will replace the declaration on the setter on line 371, which specifies the default value. If you remove the annotation on line 381 it should work.

Reply:
Thank you.